bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -389,6 +391,7 @@ export class LegendView extends AnnotationView {
       }
       `)
     }
+    // TODO background_hatch
```

Comment:
This will have to be figured out in general as currently we don't have a method to translate hatch visuals to CSS (issue https://github.com/bokeh/bokeh/issues/14312).

Reply:
Could leave the issue link in these todos